cmd/zetae2e/local/local.go

```diff
@@ -317,6 +317,7 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 			e2etests.TestBitcoinWithdrawP2WSHName,
 			e2etests.TestBitcoinWithdrawMultipleName,
 			e2etests.TestBitcoinWithdrawRestrictedName,
+			//e2etests.TestBitcoinWithdrawRBFName, // leave it as the last BTC test
```

Comment:
Why is it disabled for now?

Reply:
only RBF test requires setting `minTxConfirmations = 1` to fake and simulate a pending outbound in mempool. All other tests are supposed to be ran under `minTxConfirmations = 0`, which is the case for mainnet/testnet.